debian/ubuntu-advantage-tools.postinst

```diff
@@ -473,6 +474,10 @@ case "$1" in
       if dpkg --compare-versions "$PREVIOUS_PKG_VER" lt "29~"; then
           rename_gpg_keys
       fi
+
+      if dpkg --compare-versions "$PREVIOUS_PKG_VER" lt "31~"; then
```

Comment:
Is this `31~` version check correct? The postinst-migrations.sh file starts with
`if dpkg --compare-versions "$PREVIOUS_PKG_VER" lt "32~"; then`

Also, we now have 31.2.3 as the last 31.2.x version, so none of the above will trigger if upgrading from 31.2.3, right?

Reply:
Ohh, I see the light now. Smart.